src/common/image.c

```diff
@@ -1328,6 +1328,62 @@ static uint32_t dt_image_import_internal(const int32_t film_id, const char *file
 
   // read all sidecar files
   dt_image_read_duplicates(id, normalized_filename);
```

Comment:
Instead of adding a "duummy" record and see after if we have or not imported some duplicates. Can't we just have dt_image_read_duplicate() returns the number of duplicates found and handled?

And then, the code changed below would certainly be easier.